KarhooUISDK/Screens/QuoteList/QuoteListPresenter.swift

```diff
@@ -29,6 +29,8 @@ final class KarhooQuoteListPresenter: QuoteListPresenter {
     // TODO: REVERT TO 120 AFTER TESTS
     private let minimumAcceptedValidityToQuoteRefresh: TimeInterval = 15 // 120
     var isSortingAvailable: Bool = true
+    private var passengerCount: Int = 1
+    private var luggageCount: Int = 0
```

Comment:
This values (`passengerCount` and `luggageCount`) should not be handled by `QuoteListPresenter` directly. It's better to extend functionality of `QuoteFilterHandler`. Change `QuoteFilterHandler's` with proper computed variable.
Additionally ,afaik, those values are set, but never read, so I guess something needs to be fixed in this topic as well.